apps/managedidentity/managedidentity_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+package managedidentity
+
+import (
+	"context"
+	"testing"
+)
+
+func fakeClient(mangedIdentityId ID, options ...Option) (Client, error) {
+	client, err := New(mangedIdentityId, options...)
+
+	if err != nil {
+		return Client{}, err
+	}
+
+	return client, nil
+}
+
+func TestManagedIdentity(t *testing.T) {
+	client, err := fakeClient(SystemAssigned())
+
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = client.AcquireToken(context.Background(), "scope", WithClaims("claim"))
+
+	if err == nil {
```

Comment:
What is the expected error?

Reply:
Right now this test class can be ignored as it is to be filled in as part of the next PR, but in general this will be an Error from the AcquireToken call, we will probably check specifics down the line as the tests are mapped out and designed. These initial tests were just to make sure the code was compiling ok